README.md

```diff
@@ -109,12 +109,13 @@ const { time, start, pause, reset, status } = useTimer({
 
 The configuration and all its parameters are optional.
 
-| Property     | Type     | Default value | Description                                                                            |
-| ------------ | -------- | ------------- | -------------------------------------------------------------------------------------- |
-| endTime      | number   | null          | The value for which timer stops                                                        |
-| initialTime  | number   | 0             | The starting value for the timer                                                       |
-| interval     | number   | 1000          | The interval in milliseconds                                                           |
-| onTimeOver   | function |               | Callback function that is called when time is over                                     |
-| onTimeUpdate | function |               | Callback function that is called when time is updated                                  |
-| step         | number   | 1             | the value to add to each increment / decrement                                         |
-| timerType    | string   | "INCREMENTAL" | the choice between a value that increases ("INCREMENTAL") or decreases ("DECREMENTAL") |
+| Property        | Type     | Default value | Description                                                                            |
+| --------------- | -------- | ------------- | -------------------------------------------------------------------------------------- |
+| endTime         | number   | null          | The value for which timer stops                                                        |
```

Comment:
I think `endTime` should be `null` by default in `INCREMENTAL` mode and `0` in `DECREMENTAL` mode

Reply:
Why not... Maybe we could check this point in an other PR, as it's not related :-)